src/main/java/com/googlecode/jsonrpc4j/IJsonRpcClient.java

```diff
@@ -36,7 +36,7 @@
 	 * Invokes the given method with the given argument.
 	 *
 	 * @param methodName the name of the method to invoke
-	 * @param arguments the arguments to the method
+	 * @param argument the arguments to the method
```

Comment:
Good point. I was just trying to make JavaDoc warnings/errors go away. I'll fix that too and push a new commit.